internal/app/machined/pkg/controllers/k8s/kubelet_spec.go

```diff
@@ -243,6 +262,7 @@ func NewKubeletConfiguration(cfgSpec *k8s.KubeletConfigSpec) (*kubeletconfig.Kub
 		Kind:       "KubeletConfiguration",
 	}
 	config.StaticPodPath = constants.ManifestsDirectory
```

Comment:
we should remove/disable the static pod path.

we need to think about the upgrade sequence here, probably it should be new machine config value protected by the version contract

Reply:
in fact, this URL can be put in another controller, which builds `KubeletConfig`